src/lib/Libecl/ecl_verify_values.c

```diff
@@ -1999,3 +1999,52 @@ verify_value_preempt_sort(int batch_request, int parent_object, int cmd,
 
 	return 0;
 }
+
+/**
+ * @brief
+ *	Wrapper function for pbs_verify_attributes. Even if one server is down it contacts another
+ *	server for the verification of attributes.
+ *
+ * @see verify_attributes
+ *
+ * @param[in]	fd		-	connection socket
+ * @param[in]	batch_request	-	Batch Request Type
+ * @param[in]	parent_object	-	Parent Object Type
+ * @param[in]	cmd		-	Command Type
+ * @param[in]	attribute_list	-	list of attributes
+ *
+ * @return	int
+ * @retval 	0 - No failed attributes
+ * @retval 	+n - Number of failed attributes (pbs_errno set to last error)
+ * @retval 	-1 - System error verifying attributes (pbs_errno is set)
+ *
+ * @par		Side effects:
+ *		Modifies the TLS data for this thread\n
+ *		pbs_errno is set on encourtering error
+ *
+ * @par MT-safe: Yes
+ */
+int
+pbs_verify_attributes_wrapper(int fd, int batch_request,int parent_object, int cmd, struct attropl *attribute_list)
```

Comment:
pbs_verify_attributes() actually does not connect to any server at all. It takes the fd only to store the error code in it. So do we really need to loop over all the server instances to verify the same attribute list again and again?

Reply:
You are right. I had placed this to make sure that IFL does not proceed without making sure all connections are up as qmgr stays connected. But qmgr gets disconnected if one of the connection breaks anyway.